pom.xml

```diff
@@ -92,9 +92,9 @@
         <accumulo.version>1.8.1</accumulo.version>
         <zookeeper.version>3.4.10</zookeeper.version>
         <thrift.version>0.9.3</thrift.version>
-        <hadoop.version>2.6.4</hadoop.version>
+        <hadoop.version>2.7.3</hadoop.version>
         <spark.version>2.3.1</spark.version>
-        <hbase.version>1.3.1</hbase.version>
+        <hbase.version>2.0.2</hbase.version>
```

Comment:
looks like 2.0.6 is out, could we go to that version? do you know if the code would work with 2.1 or 2.2 at runtime?